src/scalarstats.jl

```diff
@@ -226,6 +226,164 @@ returns a vector of quantiles, respectively at `[0.0, 0.2, 0.4, 0.6, 0.8, 1.0]`.
 """
 nquantile(x, n::Integer) = quantile(x, (0:n)/n)
 
+"""
+    quantilerank(itr, value; method=:inc)
+
+Compute the quantile position in the [0, 1] interval of `value` relative to collection `itr`.
+
+Different definitions can be chosen via the `method` keyword argument.
+Let `count_less` be the number of elements of `itr` that are less than `value`, 
+`count_equal` the number of elements of `itr` that are equal to `value`, `n` the length of `itr`, 
+`greatest_smaller` the highest value below `value` and `smallest_greater` the lowest value above `value`. 
+Then `method` supports the following definitions:
+
+- `:inc` (default): Return a value in the range 0 to 1 inclusive. 
+Return `count_less / (n - 1)` if `value ∈ itr`, otherwise apply interpolation based on 
+definition 7 of quantile in Hyndman and Fan (1996)
+(equivalent to Excel `PERCENTRANK` and `PERCENTRANK.INC`).
+
```

Comment:
So you'd say something like this is correct?
```suggestion
This definition corresponds to the lower semi-continuous inverse of
[`quantile`](@ref) with its default parameters.

```

Reply:
yes - from a practical perspective (although strictly speaking the `quantile` function does not have an inverse in general)